lesson02/assignment-alt/gulpfile.js

```diff
@@ -0,0 +1,68 @@
+const {
```

Comment:
Remember, your lesson3 code should go in your lesson3 assignment folder. Please watch for this in future assignments